openapi/evidence/openapi.yaml

```diff
@@ -109,3000 +109,27 @@ paths:
                                 $ref: '#/components/schemas/Status'
 components:
     schemas:
-        ABAC:
```

Comment:
This looks fishy. Why did this PR delete the generated ontology structures for OpenAPI?

Reply:
I fixed it by removing the Evidence class from the ontology.owx file.